CHANGELOG.md

```diff
@@ -20,7 +20,19 @@ The headers used in [Keep a Changelog](http://keepachangelog.com) are:
 - Fixed - for any bug fixes.
 - Security - in case of vulnerabilities.
 
-# [Unreleased](https://github.com/puppetlabs/beaker/compare/4.27.0...master)
+# [Unreleased](https://github.com/puppetlabs/beaker/compare/4.28.0...master)
+
```

Comment:
This looks odd to include in the actual gem that's pushed to Rubygems. I won't block on this since it looks like this was done in the past as well, but it's not something we do anywhere else AFAIK.

Reply:
ages ago, when we wrote changelogs per hand, this was common. Personally I never used the link and it was often forgotten to update it. I didn't want to rewrite the whole changelog to our changelog-generator, maybe that's something for the next release.